src/components/radio-group/radio-group.tsx

```diff
@@ -0,0 +1,45 @@
+import { Component, Host, h, Prop, Watch, State } from "@stencil/core";
+
+@Component({
+  tag: "radio-group",
```

Comment:
As a best practice, component tags should have a namespace (e.g., `"anveshmekala-radio-group"`).